recipe/meta.yaml

```diff
@@ -39,22 +39,22 @@ requirements:
   build:
     - 7za               # [win]
     - bzip2 1.0.*       # [unix]
-    - curl              # [win]
+    - curl >=7.44.0,<8              # [win]
     - openssl 1.0.*     # [unix]
-    - readline 6.2*     # [unix]
-    - sqlite 3.13.*     # [unix]
-    - tk 8.5.*          # [unix]
+    - readline 7.0     # [unix]
+    - sqlite 3.20.*     # [unix]
+    - tk 8.6.*          # [unix]
```

Comment:
So should [`tk` in the pinnings]( https://github.com/conda-forge/conda-forge.github.io/blob/24a34313f341c352ea87806b2dc63de672ff881a/scripts/pin_the_slow_way.py#L98 ) be updated as it points to `8.5.*` currently.